src/workerd/util/strings.c++

```diff
@@ -98,4 +98,17 @@ kj::Array<kj::byte> stripInnerWhitespace(kj::ArrayPtr<kj::byte> input) {
   return result.first(len).attach(kj::mv(result));
 };
 
+bool strcaseeq(kj::ArrayPtr<const char> a, kj::ArrayPtr<const char> b) {
```

Comment:
this can be significantly improved by reducing the branches and using lookup tables etc. can you add a todo?